Tetris support

udibr opened the pull request:
I've added two changes I was missing (whem moving from master to dev)
hope it is working now.

mgbellemare commented on src/games/supported/Tetris.cpp:
```diff
@@ -0,0 +1,133 @@
+    int y = readRam(&system, 114);
+    reward_t score = 1 * (x & 0x000F) + 10 * ((x & 0x00F0) >> 4) + 100 * (y & 0x000F);
+    if (score > m_score) {
+        std::cerr << "Score=" << score << std::endl;
```
Please remove the printout.

mgbellemare commented:
Tetris support looks good, save for the few comments I made in the other PR (please address).

udibr commented:
(hopefully) I addressed all the issues regarding Tetris. If you want it to be called Tetris26 and not Tetris then I need to change the class names etc...

mgbellemare commented:
LGTM. One thing I noticed is that the tetromino sequence is fixed and cyclical. I suspect randomly setting the CPU registers doesn't fix this beyond randomizing where you start in the sequence.
